dotcom-rendering/src/lib/renderElement.tsx

```diff
@@ -474,7 +474,7 @@ export const renderElement = ({
 				<NumberedTitleBlockComponent
 					position={element.position}
 					html={element.html}
-					format={element.format}
+					format={format}
```

Comment:
Changing the type from `FEFormat` to `ArticleFormat` so that we can see the `palette` changes in the stories. Previously format was ignored.